packages/@aws-cdk/aws-ivs/README.md

```diff
@@ -10,11 +9,76 @@
 > [CFN Resources]: https://docs.aws.amazon.com/cdk/latest/guide/constructs.html#constructs_lib
 
 ---
-
 <!--END STABILITY BANNER-->
 
+Amazon Interactive Video Service (Amazon IVS) is a managed live streaming
+solution that is quick and easy to set up, and ideal for creating interactive
+video experiences. Send your live streams to Amazon IVS using streaming software
+and the service does everything you need to make low-latency live video
+available to any viewer around the world, letting you focus on building
+interactive experiences alongside the live video. You can easily customize and
+enhance the audience experience through the Amazon IVS player SDK and timed
+metadata APIs, allowing you to build a more valuable relationship with your
+viewers on your own websites and applications.
+
 This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
 
+### Channels
+
+An Amazon IVS channel stores configuration information related to your live
+stream. You first create a channel and then contribute video to it using the
+channel’s stream key to start your live stream.
+
+You can create a channel
+
+```ts
+const myChannel = new ivs.Channel(this, 'Channel');
+```
+
+#### Importing an existing channel
+You can reference an existing channel, for example, if you need to create a
+stream key for an existing channel
+
+```ts
+const myChannel = ivs.Channel.fromChannelArn(this, 'Channel', myChannelArn);
+```
+
+#### Stream Keys
+A Stream Key is used by a broadcast encoder to initiate a stream and identify
+to Amazon IVS which customer and channel the stream is for. If a customer is
+storing this value, it should be treated as if it were a password.
+
+You can create a stream key for a given channel
+
+```ts fixture=with-channel
+const myStreamKey = myChannel.addStreamKey('StreamKey');
+```
+
+#### Private Channels
+
+Amazon IVS offers customers the ability to create private channels, allowing
+customers to restrict their streams by channel or viewer. Customers control
```

Comment:
are `customers` different from the `You can...` referenced in previous sections?
Can we align the terminology so it's consistent? (preference is `you` as it's used in many other
READMEs)